### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by tkachen**

### CODE REVIEW

File: locales/ru.yml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+common.save_as: Сохранить как
+common.save.success: Успешно сохранено
+common.apply: Применить
+common.apply.success: Успешно применено
+common.apply.failed: Ошибка при применении
+common.confirm.show_detail: Показать детали
+common.network.connection.error: Ошибка сетевого соединения
+common.please_quit_edit: Пожалуйста, выйдите из режима редактирования
+common.time_unit.hours: ' часов'
+common.time_unit.millisecond: миллисекунд
```
Comment (anikitin85) [inline locales/ru.yml]: Will not work with numbers ending with 1, 2, 3, and 4 ("21 миллисекунд"). Just ' мс' should be fine.

File: locales/ru.yml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+# Feedback
+feedback: Обратная связь
+feedback.title: Обратная связь
+feedback.content: Сообщение
+feedback.content.no_empty: Пожалуйста, заполните сообщение
+feedback.content.max_1000: Максимум 1000 символов
+feedback.email.label: Email (необязательный)
+feedback.email.placeholder: Пожалуйста, введите свой Email
+feedback.email.not_valid: Неправильный Email адрес
```
Comment (anikitin85) [inline locales/ru.yml]: "Неправильный адрес электронной почты" (Or if you'd like to keep "Email" in the translation, a hyphen is needed: "Неправильный Email-адрес")

File: locales/ru.yml [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+feedback.submit: Отправить
+feedback.submit.error: Ошибка при отправке. Пожалуйста, повторите попытку
+feedback.submit.success: Успешно отправлено. Нажмите, чтобы закрыть
+feedback.support_email: 'Email службы поддержки: '
+
+# Preference Window
+preferences.setting: Опции
```
Comment (anikitin85) [inline locales/ru.yml]: Translated differently frome above. Should be "Параметры" or "Настройки"

File: locales/ru.yml [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+preferences.setting.theme: Тема
+preferences.setting.theme.dark: Темная
+preferences.setting.theme.light: Светлая
+preferences.setting.update_channel: Пакеты обновлений
+preferences.setting.update_channel.beta: Бета-версии
+preferences.setting.update_channel.latest: Релизные версии 
+preferences.setting.animation: Анимация на стартовом экране
+preferences.setting.animation.on: Вкл.
+preferences.setting.animation.off: Выкл.
+preferences.setting.x_action: Действии при закрытии главного окна
```
Comment (anikitin85) [inline locales/ru.yml]: A typo. Should be "действиЕ..."

File: locales/ru.yml [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+preferences.setting.minimize_to_tray: Свернуть в трей
+preferences.setting.quit_program: Выйти из программы
+preferences.setting.auto_launch: Запуск при авторизации
```
Comment (anikitin85) [inline locales/ru.yml]: "Запускать при входе в систему"

File: locales/ru.yml [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+preferences.setting.auto_launch.yes: Да
+preferences.setting.auto_launch.no: Нет
+preferences.update: Обновления
+preferences.update.check: Проверить наличие новой версии
```
Comment (anikitin85) [inline locales/ru.yml]: "Проверить обновления"

File: locales/ru.yml [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+kbd.general.info.battery: Батарея
+kbd.general.info.running_time: Время работы
+kbd.general.setting: Опции
+kbd.general.setting.layout: Раскладка
+kbd.general.setting.tap_layout: Режим Tap
+kbd.general.setting.magic_fn: Клавиша Magic Fn
+kbd.general.setting.macro_trigger: Макросы
+kbd.general.setting.capslock_led: Индикатор CapsLock
+kbd.general.setting.auto_sleep: Автовыключение
+kbd.general.setting.options.disable: Отключить
```
Comment (anikitin85) [inline locales/ru.yml]: Выкл.

File: locales/ru.yml [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+kbd.general.setting.options.enable: Включить
```
Comment (anikitin85) [inline locales/ru.yml]: Вкл.

File: locales/ru.yml [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+kbd.firmware.list: Список прошивок
+kbd.firmware.annepro1.download.firmware: Скачать прошивку
+kbd.firmware.annepro1.download.latest_firmware: Скачать актуальную прошивку
+kbd.firmware.annepro1.download.dfu_tool: Скачать DFU Tool
+kbd.firmware.find_new_version: 'Найдена новая версия: '
+kbd.firmware.no_new_version: Текущая прошивка наиболее актуальная
+kbd.fireware.upgrade.btn: Обновить
+kbd.firmware.upgrade.btn.to_latest: Обновить до последней версии 
+kbd.firmware.upgrade.title: Обновление прошивки
+kbd.firmware.upgrade.content: Пожалуйста, не отсоединяйте USB кабель в процессе обновления
```
Comment (anikitin85) [inline locales/ru.yml]: 1. A hyphen needed: "... USB-кабель ..."
2. "Пожалуйста" can be omitted

File: locales/ru.yml [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+kbd.firmware.upgrade.status.restart: Перезапуск клавиатуры...
+kbd.firmware.upgrade.status.success: Прошивка успешно обновлена.
+
+# Main Window, LEDs
+kbd.led: Подсветка
+kbd.led.effect.group: Группы эффектов
+kbd.led.effect.group.reset: Сбросить
+kbd.led.effect.group.delete.confirm_content: Вы уверены, что хотите удалить эту группу?
+kbd.led.effect.list: Список эффектов
+kbd.led.effect.list.limit.annepro1_three: Максимальное количество эффектов для одной группы - 3
```
Comment (anikitin85) [inline locales/ru.yml]: An em dash should be used instead of the hyphen: "...группы — 3"

File: locales/ru.yml [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+kbd.led.effect.list.limit.annepro2_sixteen: Максимальное количество эффектов для одной группы - 16
```
Comment (anikitin85) [inline locales/ru.yml]: An em dash should be used instead of the hyphen: "...группы — 16"

File: locales/ru.yml [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+kbd.keycode_group.media: Медиа
+kbd.keycode_group.keypad: Цифровой блок
+kbd.keycode_group.other: Другое
+# Key tooltips
+icon.left_gui: Левый Win/Cmd
+icon.right_gui: Правый Win/Cmd
+icon.mute: Отключить звук
+icon.volume.up: Повысить громкость
+icon.volume.down: Понизить громкость
+icon.bl.toggle: Вкл./Выкл. подсветку
```
Comment (anikitin85) [inline locales/ru.yml]: The second word should be lowercased: "Вкл./выкл...."

**Comment by tkachen** [comment]
@anikitin85 Thank you for all suggestions and fixes!

**Comment by obinslab** [comment]
Thanks all of you

**Comment by obinslab** [comment]
We had a activity on Twitter: https://twitter.com/obinslab/status/1119572238854803457
We want to send you a ANNE PRO 2 as a gift.
Please contact us by email, our email is:service@obins.net

Best regards!

**Pull request merged**